README.md

```diff
@@ -0,0 +1,13 @@
+# __staff-management-api__
+An opensource api which provides various options to manage office/household help/staff.
+
+##__Steps to setup project__
+
+1. Go to https://start.spring.io and create a project with name _staff-management-api_
```

Comment:
Here you should be explaining the steps to set up and run your repo. not the steps to create a new SB project.